### ISSUE TITLE
QSelect keyboard navigation bugs (while using tabindex)

### ISSUE THREAD

**Issue by Peter-Krebs** [quasarframework/quasar#5482]
**Description**
QSelect has unintended behaviour when using keyboard navigation. I have noticed this while using `tabindex` attributes.  
Arrow keys don't open the options. Pressing tab after a selection moves to the next QSelect instead of the next input. I have noticed the value disappear after selecting it with the mouse and then pressing tab.

See _Codepen_ and _To Reproduce_ below.

**Codepen**
https://codepen.io/peter-krebs/pen/yLLKrgY

**To Reproduce**

_Arrow Down:_  
1. Focus the text input
2. Press `tab` (the QSelect below should look focused now)
3. Pressing `down arrow` / `enter` doesn't open the options

_Chosen value disappears:_
1. Open the options of a QSelect by using the mouse
2. Choose a value by klicking
3. Press `tab`
4. Value disappears / reverts to the previous value

_Tab focuses next select:_
1. Follow "Chosen value disappears" above
2. The next form element focused is not the QInput following, it is the next QSelect

**Expected behavior:**
- Pressing `down arrow` / `enter` while the QSelect is focused should open the options
- The value should be kept even with mixed mouse/keyboard navigation
- When a QSelect is focused and then pressing `tab`, the next form element focused should be the next in line (either naturally or by tabindex), not the next QSelect

**Platform:**
OS: Windows 10 64bit
Node: 10.15.3
NPM: 6.4.1
Browsers: Chrome 77.0.3865.120 64bit
iOS: not tested
Android: not tested
Electron: not tested

**Comment by rstoenescu** [comment]
Fix will be available in "quasar" v1.4.0.

**Comment by Peter-Krebs** [comment]
Thanks for addressing the issue quickly - will test v1.4.0 when it's out
